docs/src/Upsell.doc.js

```diff
@@ -22,133 +41,370 @@ card(
+            primaryAction={{label: "Install now"}}
+            dismissButton={{
+              accessibilityLabel: 'Dismiss banner',
+              onDismiss: ()=>{},
+            }}
+            imageData={{
+              component: <Icon icon="ads-stats" accessibilityLabel="" color="darkGray" size={32}/>
+            }}
+          />
+        </Flex>
+        `}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection name="Accessibility">
+    <MainSection.Subsection
+      title="Labels"
+      description={`
+      \`dismissButton\`, \`primaryAction\`, and \`secondaryAction\` each require a short, descriptive label for screen readers, should also be localized.
+
+      In the case of [Buttons](/Button), alternative text should be provided to replace vague text like "Visit" or "Learn more" with more descriptive information, like "Learn more about work from home resources".
+
+      For the \`dismissButton\` [IconButton](/IconButton), the label provided should indicate the intent, like “Dismiss this banner”.
+
+      The [Image](/Image) or [Icon](/Icon) supplied to \`imageData\` should only supply an \`alt\` or \`accessibilityLabel\`, respectively, if the Image or Icon supplies extra context or information. Icons in Upsells are often purely decorative, and can therefore have an empty string as the \`accessibilityLabel\`.
+      `}
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
 <Upsell
-  message="Single line upsell with no title or call to action."
+  title="Give $30, get $60 in ads credit"
+  message="Earn $60 of ads credit, and give $30 of ads credit to a friend"
+  primaryAction={{href: "https://pinterest.com", label: "Send invite", accessibilityLabel: "Invite friend to use ads", target: "blank"}}
   dismissButton={{
     accessibilityLabel: 'Dismiss banner',
     onDismiss: ()=>{},
   }}
+  imageData={{
+    component: <Icon icon="pinterest" accessibilityLabel="" color="darkGray" size={32}/>
+  }}
 />
-  `}
-  />,
+        `}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
 );
 
 card(
-  <Example
-    name="Icon"
-    defaultCode={`
+  <MainSection
+    name="Localization"
+    description={`Remember to localize all link or button labels, as well as \`title\` and \`message\`.`}
+  >
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+<Upsell
+  imageData={{
+    component: <Icon icon="send" accessibilityLabel="" color="darkGray" size={32}/>
+  }}
+  title="Fast fertig! Beenden Sie die Installation Ihres Pinterest-Tags und erhalten Sie ein Guthaben von 10 Euro"
+  message="Verfolgen Sie die Anzeigenkonvertierung - Umsatz, Traffic und mehr - mit dem Pinterest Tag"
+  primaryAction={{label: "Beanspruche jetzt", accessibilityLabel: "Beanspruche Guthaben jetzt", target: "blank"}}
+/>
+        `}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection name="Variants">
```

Comment:
Any good use case for this? When do we wanna use only-texts Upsells? It defeats the purpose of `promoting an action or upgrading something`. But there might be a good use case.

Reply:
@ashleyseto do you know of any product examples where Upsell is used without a title or actions? I only have dummy text here right now

Reply:
(clarified in slack)